code-challenges.test.js

```diff
@@ -29,12 +64,32 @@ const secretCodeWord3 = "Eccentric"
 
 // a) Create a test with expects statement using the variable provided.
 
-const fruitArray = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach", "Kiwi"]
+describe("findWords", () => {
+    it("takes in an array of words and a single letter and returns an array of all the words containing that particular letter", () => {
+        const fruitArray = ["Mango", "Cherry", "Apricot", "Blueberry", "Peach", "Kiwi"]
+        
+        const filterLetterA = "a"
+        // Expected output: ["Mango", "Apricot", "Peach"]
+        const filterLetterE = "e"
+        // Expected output: ["Cherry", "Blueberry", "Peach"]
+        
+        expect(findWords(fruitArray, filterLetterA)).toEqual(["Mango", "Apricot", "Peach"])
+        expect(findWords(fruitArray, filterLetterE)).toEqual(["Cherry", "Blueberry", "Peach"])
+        
+    })
+}) 
+
+const findWords = (words, letter) => {              // create a function that takes in an array and a letter as parameters
+    const filteredWords = []                        // create an empty array to push into
+    for(let i = 0; i < words.length; i++) {         // create for loop
+        if(words[i].includes(letter)) {             // check if current word includes the letter
+            filteredWords.push(words[i])            // push word into new array 
```

Comment:
Same thing here, how can you do this without the empty variable `filteredWords`? What can you use that words on an array and returns an array on its own?